Cargo.toml

```diff
@@ -63,4 +63,5 @@ protobuf = { git = "https://github.com/pingcap/rust-protobuf", rev = "82b49fea7e
 protobuf-codegen = { git = "https://github.com/pingcap/rust-protobuf", rev = "82b49fea7e696fd647b5aca0a6c6ec944eab3189" }
 
 [workspace]
-members = [ "stress" ]
+members = ["stress", "ctl"]
+default-members = ["ctl"]
```

Comment:
Probably this line is wrong. This means default use `ctl` instead of `raft_engine` crate, and the CI fails to locate tests.

Reply:
if we do no put `ctl` in default-members,  run `cargo build` in root will not compile `ctl` like `stress`, is that OK? @tabokie

Reply:
Yes